.github/workflows/molecules.yml

```
@@ -46,18 +59,17 @@ jobs:
     - name: Determine Istio version to use
       id: determine-istio-version-to-use
       run: |
-        if [[ "${{ github.event_name }}" == "workflow_dispatch" && -n "${{ github.event.inputs.istio_minor_version_offset }}" ]]; then
-          OFFSET="${{ github.event.inputs.istio_minor_version_offset }}"
-          # Convert to absolute value - we want a positive offset, though some people might find it more intuitive to say "-1" offset for the previous version
-          OFFSET=$(( OFFSET < 0 ? -OFFSET : OFFSET ))
-        else
+        OFFSET="${{ inputs.istio_minor_version_offset }}"
```

Can you put this conversion to absolute value back in? This was something me and Jay went back and forth on (should a user specify the offset version as a negative or a positive number?)  Rather than pick one, we just added this so it accepts both. This way a user doesn't have to worry about it - just specify what is intuitive to them and it will work.

```suggestion
        OFFSET="${{ inputs.istio_minor_version_offset }}"
        # Convert to absolute value - we want a positive offset, though some people might find it more intuitive to say "-1" offset for the previous version
        OFFSET=$(( OFFSET < 0 ? -OFFSET : OFFSET ))```